WordPress/gutenberg
# [RNMobile] Set tintColor for ActionSheetIOS

## Related PRs

* `gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5074

## What?

This PR explicitly sets a `tintColor` on the [ActionSheetIOS](https://reactnative.dev/docs/actionsheetios) component.

## Why?

At the moment, the `tintColor` is not explicitly defined and it's easy for it be unintentionally overridden by host apps. Explicitly setting the value will avoid any unintentional overides.

## How?

The [ActionSheetIOS](https://reactnative.dev/docs/actionsheetios) component's `tintColor` prop is used in combination with the `usePreferredColorScheme` hook to set different colours for light/dark mode. 

## Testing Instructions

1. Navigate to the editor in the iOS WordPress app.
2. To access the ActionSheetIOS component, add an image block and tap the option to `add image`. Another option for accessing the component is to tap the ellipsis in the bottom right of a block for the actions menu.
3. Confirm that this PR doesn't introduce any unexpected colour changes in the WordPress app.

## Screenshots or screencast 

| Light  | Dark |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/182964585-dfbc9d30-8c2e-4bcc-aaac-cdd7af9bdade.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/182964695-0915c990-36cb-40b9-be9e-fd89ef179d93.png" width="100%"> |
